machines/store.ts

```diff
@@ -153,12 +164,26 @@ export const storeMachine =
                 sendUpdate(),
               ],
             },
+            TAMPERED_VC: {
+              actions: ['setIsTamperedVc'],
+            },
+            RESET_IS_TAMPERED: {
+              actions: ['resetIsTamperedVc'],
+            },
           },
         },
       },
     },
     {
       actions: {
+        setIsTamperedVc: model.assign({
+          isTampered: (_, event) => true,
+        }),
+
+        resetIsTamperedVc: model.assign({
+          isTampered: (_, event) => false,
```

Comment:
can we remove the unused parameters ( _, event )

Reply:
Refactored.